src/index.html

```diff
@@ -12,7 +12,15 @@
 
     <base href="/">
 
-    <link rel="icon" href="img/favicon.ico" size="16x16 32x32 48x48 64x64" type="image/vnd.microsoft.icon">
+    <link rel="apple-touch-icon" sizes="180x180" href="/img/apple-touch-icon.png">
+    <link rel="icon" type="image/png" href="/img/favicon-32x32.png" sizes="32x32">
+    <link rel="icon" type="image/png" href="/img/favicon-16x16.png" sizes="16x16">
+    <link rel="manifest" href="/img/manifest.json">
```

Comment:
Let's move `manifest.json` out of the `img` folder. I think its conventional to have it at the root next to things like `robots.txt`, and that would be better than in the `img` folder, anyway. While doing this, we'll need to modify the `build-other`/`watch-other` npm scripts to also copy over JSON files as well. New NPM scripts should be:

```
"build-other": "cpx \"src/**/*.{txt,json}\" dist --verbose"
"watch-other": "cpx \"src/**/*.{txt,json}\" dist --watch --verbose"
```

Only difference in the scripts is the brace expansion in the source glob.